src/pages/specimen-collection.js

```diff
@@ -357,7 +359,6 @@ const SpecimenCollection = () => {
                             value="morning"
                             id="morning"
                           />
-                          <RadioButton labelText="N/A" value="n_a" id="n_a" />
```

Comment:
We also need to set the default value of this field to null since the "N/A" option is removed. This means we will need to handle the error message like we did above for specimen collection

Reply:
Nice removing this one but how about the error handling?

Reply:
And schema update?